### ISSUE TITLE
Weird state returns stash.go:106

### ISSUE THREAD

**Issue by nippe** [otm/git-build-state#1]
Merged a pull request with --squash to master.

Tried git build-state and get: `stash.go:106`

If I fire it of with the --log flag I get can see the build:
```
> git build-state --log
<commit-hash> Merge pull request #2 in PROJECT/repositrory from feature/branch-name to master
   Successful: 0, In Progress: 0, Failed: 0
```

Can it be that it has now state according to above? 0 on all states...

**Comment by otm** [comment]
Look at the build-state in stash, does it look OK?

It looks like there was a request made to Stash to retrieve the build state and that returned an empty body. Which is a bit weird. Does it work in other repositories?

**Comment by nippe** [comment]
It works for other repositories under another project. Digging around it looks like they have different permissions and access keys set up on the project level. Donno but that might have something to do with it.

**Comment by otm** [comment]
You could try making request to Stash manually. To see what is going on with Stash.

`curl -u username:password stash.example.com/rest/build-status/1.0/commits/<commitid> `

**Comment by otm** [comment]
Fixed in 30b1b31 available in release v0.0.4

**Comment by nippe** [comment]
👍
